AU/Public/Update-Package.ps1

```diff
@@ -223,7 +223,7 @@ function Update-Package {
         $script:is_forced = $false
         if ([AUVersion] $Latest.Version -gt [AUVersion] $Latest.NuspecVersion) {
             if (!($NoCheckChocoVersion -or $Force)) {
-                if ( !$au_GalleryUrl ) { $au_GalleryUrl = 'https://chocolatey.org' } 
```

Comment:
I have mine patched like:

`if ( $env:au_GalleryUrl ) {$au_GalleryUrl = $env:au_GalleryUrl} else { $au_GalleryUrl = 'https://chocolatey.org' }`
because L227 uses the var too

Reply:
Excuse my ignorance, but I don't see what difference this makes. Our two implementations appear to do the exact same thing, except that yours checks for the presence of a variable and mine checks for the absence of a variable.

Reply:
The difference is that users don't have to think about what kind of variable is there as both will work.